Tests/MapboxSearchUITests/MockServerUITestCase.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 
-class MockServerUITestCase: BaseTestCase {
-    let server = MockWebServer()
+class MockServerUITestCase<Mock: MockResponse>: BaseTestCase {
+    typealias Mock = Mock
```

Comment:
nit: can we remove typealias?

Reply:
Oh the MockServerUITestCase _doesn't_ have `typealias Mock = Mock` because all the usages are generics with `server` functions or properties so the generic is inferred and the type is available. 

For example in this usage `setResponse` is typed to accept a `Mock` generic and that is provided by the VisibilityTestCase subclass so we don't need a typealias for Mock.

https://github.com/mapbox/mapbox-search-ios/blob/051ef74e618b4b60dd7d11f94ee78d32cba375f1/Tests/MapboxSearchUITests/VisibilityTestCase.swift#L3-L9

Reply:
Well MockServerUITestCase did have a typealias, but does not need it (as explained above). Will remove from UITestCase!